src/shared/navigation/read-query-params.js

```diff
@@ -0,0 +1,11 @@
+import { parse } from 'query-string'
```

Comment:
I think https://www.npmjs.com/package/use-query-params is better for our use-case in this app.

Reply:
I think @HendrikThePendric considered it but then switch to this one as the values are needed for constructing/manipulating the central state, which is outside of the react scope. I can investigate this a bit further and will get back to you